### ISSUE TITLE
Reduce commit search results

### ISSUE THREAD

**Issue by ScriptShot** [ForkIssues/TrackerWin#418]
Hi,
at the moment, hits are highlighted. With many hits that have been comitted over a long time, the cherry-picking option is very tedious. Maybe you can make it configurable, that instead of the highlighting all other entries of the history are filtered out.

That would be really great :)

**Comment by ScriptShot** [comment]
Hi. Is it possible that this will be implemented?

**Comment by DanPristupov** [comment]
It's possible and is in our todo list, but we don't have an ETA yet. Should appear in a few months probably.

**Comment by Otiel** [comment]
Duplicate with #403?

**Comment by DanPristupov** [comment]
@Otiel no, it's not. #403 proposes to create a separate view, which is different.

**Comment by Otiel** [comment]
I see. You consider to implement #418 as a "simple" boolean option for the search (_highlight_ vs _filter_), is that so?

**Comment by DanPristupov** [comment]
Yes, this is the idea. I think we can add a filter toggle button in the search bar which will hide the commits that don't match. This will allow us to have features like 'filter commits by author', etc.

We didn't do a proof-of-concept demo though.

**Comment by michael-hawker** [comment]
Yeah, it'd be nice if there was a toggle to filter the search list just down to the found results.

**Comment by DanPristupov** [comment]
Now the sidebar search shows only the matches.
